sdk/communication/Azure.Communication.Chat/api/Azure.Communication.Chat.netstandard2.0.cs

```diff
@@ -157,9 +165,24 @@ public partial class CreateChatThreadResult
         public Azure.Communication.Chat.ChatThreadProperties ChatThread { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.Communication.Chat.ChatError> InvalidParticipants { get { throw null; } }
     }
+    public partial class SendChatMessageOptions
+    {
+        public SendChatMessageOptions() { }
+        public Azure.Communication.Chat.ChatMessageType ChatMessageType { get { throw null; } set { } }
```

Comment:
You don't have to prefix members with "Chat". The "no ambiguous identifiers" guideline applies to type names only. And so MessageType as a name of the property would be good.